node/in_request.js

```diff
@@ -69,6 +69,24 @@ inherits(TChannelInRequest, EventEmitter);
 
 TChannelInRequest.prototype.type = 'tchannel.incoming-request';
 
+TChannelInRequest.prototype.extendLogInfo = function extendLogInfo(info) {
+    var self = this;
+
+    // TODO: add:
+    // - request id?
+    // - tracing id?
+    // - other?
+    info.inRemoteAddr = self.remoteAddr;
+    info.serviceName = self.serviceName;
+    if (self.endpoint !== null) {
```

Comment:
Perhaps `info.arg1 = null` up front to improve probability of producing a consistent hidden class.